posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+The first Community Call took place last week on the 26th of January. You can [listen back to the full Space](https://twitter.com/Waku_org/status/1750927368644919722), and read the highlights below. 
+
+[Vaclav](https://twitter.com/vpavlin), [Guru](https://twitter.com/hackyguru), and [Aaron](https://twitter.com/vrycmfy) from the Waku team participated in the call. 
+
+
+### Waku network updates 
+
+**Vaclav**: If you don't know what the Waku Network is, it is our attempt to formalise a network of peer-to-peer nodes that would form some network running Waku protocols. We are very proud to have released the Gen 0, and there will likely be more generations to come right before Devconnect in Istanbul this November.
+
+It's just the beginning for the Waku Network; we're still dogfooding it. You can see [the list of milestones and the roadmap for improvements](https://github.com/waku-org/pm/issues/101) put forward by the research team. There’s also an interesting discussion you can join in the [Vac Forum](https://forum.vac.dev/t/message-validation-marketplace-waku-network-of-networks/262/2).
+
+As part of Generation 0 of Waku Network, we have integrated sharding, ensuring that it’s scalable to thousands of users and that the bandwidth is reasonable for people running on Raspberry Pis or smaller devices with weaker connectivity. Our client targeted at low-resource devices, JSWaku, has been making progress towards fully supporting sharding and, by that, supporting the full Waku Network.
+
+
+### Future incentivisation for running Waku nodes?
+
+**Vaclav: **We are often asked about how we incentivise node runners, as well as who is running the nodes and why. There are currently no monetary or token incentives. However, the research team has recently published findings that offer a glimpse into an incentivisation proposal. That's very interesting and promising for the future of Waku node operators to maybe get something in return.
+
+
+### Growth of the Waku dev ecosystem
+
+**Guru: **We had our first Waku hackathon at ETHRome last year where 120 hackers participated. We received eight project submissions from EthRome, which was a high number for us as a first-time sponsor. At ETHIstanbul our submissions increased to 20. We closed the year with ETHIndia where we had a huge total of 51 project submissions.
+
+The graph has been trending consistently upwards as more hackers and developers discover Waku. We are exploring ways to offer support to these projects in terms of development, as well as raising funding and grants.
+
+
+### Project integrations with Waku
+
+**Guru: **Waku has been integrated into several existing projects already. [Status](https://status.app/), of course, is the original use case for Waku, and they have been the first to benefit from the Waku Network. 
```

Comment:
wellll. Status does not really the waku network. We can say that we are in the finali stages of collaborating with Status to integrate Waku's latest developments.

Reply:
updated